backend/src/openmp/projective_functor.hpp

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
Not sure about moving this to backend. Do you see a way to keep it in the core lib? Ideally core is everything octree-related but denseslam independent. Backend at this stage is somehow a mix which I'd rather avoid (think of someone that wants to use the lib without the whole denseslam dependency).